.github/workflows/dependencies-test.yaml

```
@@ -0,0 +1,175 @@
+name: Dependencies test
+
+on:
+  workflow_dispatch: # Manual trigger
+
+jobs:
+  integration:
+    strategy:
+      fail-fast: false
+      max-parallel: 1
+      matrix:
+        kubernetes:
+          - v1.17.17
+          - v1.18.19
+          - v1.19.11
+          - v1.20.7
+          - v1.21.1
+        tekton:
+          - v0.19.0
+          - v0.20.1
+          - v0.21.0
+          - v0.22.0
+          - v0.23.0
+          - v0.24.3
+          - v0.25.0
+        go-version: [1.15.x]
```

I am sorry it took me so long to look into the PR. In the meantime, we have bumped to 1.16 already.
```suggestion
        go-version: [1.16.x]
```